_toc.yml

```diff
@@ -84,6 +84,7 @@ entries:
       - file: docs/platform/howto/list-billing
         title: Billing management
         entries:
+          - file: docs/platform/howto/access-suspended-due.rst-to-past-due-invoice
```

Comment:
You might want to place it beside `docs/platform/howto/payment-issues-plan-upgrades` in the TOC

Reply:
I am pretty sure I did .rst in the end, do you know how can we rename the file ?
All suggested updates has been committed and then marked resolved, is it the correct way ? or I need to go to file again and edit manually ?